h/api/storage.py

```diff
@@ -114,15 +118,44 @@ def create_annotation(request, data):
     # annotation.updated get created.
     request.db.flush()
 
+    update_document_metadata(
+        request.db, annotation, document_meta_dicts, document_uri_dicts)
+
+    return annotation
+
+
+def update_document_metadata(session,
+                             annotation,
+                             document_meta_dicts,
+                             document_uri_dicts):
+    """
```

Comment:
I've added this function to `storage`'s public API. It is not called by any code outside of `storage`, but maybe it is a potentially useful thing to have in th